### ISSUE TITLE
could not list local ollama models

### ISSUE THREAD

**Issue by Yifei0727** [carlrobertoh/CodeGPT#609]
### What happened?

bugs only on version: 2.8.2-241.1
```idea
IntelliJ IDEA 2024.1.2 (Community Edition)
Build #IC-241.17011.79, built on May 22, 2024
Runtime version: 17.0.11+1-b1207.24 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 14.5
Kotlin analyzer version: 2.0.0-ij241.15989-9
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 10
Metal Rendering is ON
Non-Bundled Plugins:
  ee.carlrobert.chatgpt (2.8.2-241.1)
Kotlin: 241.17011.79-IJ
```
Unable list available models correctly

<img width="1436" alt="image" src="https://github.com/carlrobertoh/CodeGPT/assets/6064040/2e721b98-059c-4d91-a073-908d8d79ed63">

#### Debug

<img width="982" alt="image" src="https://github.com/carlrobertoh/CodeGPT/assets/6064040/d27f5969-fca5-472f-8dea-11a0cb304e96">

<img width="1321" alt="image" src="https://github.com/carlrobertoh/CodeGPT/assets/6064040/488975d6-38a7-493c-8556-aba765c8f5f1">

<img width="1392" alt="image" src="https://github.com/carlrobertoh/CodeGPT/assets/6064040/b7da7919-2b50-47b9-82c0-c1bf822733e6">
<img width="1408" alt="image" src="https://github.com/carlrobertoh/CodeGPT/assets/6064040/bae1bc20-af08-417b-a12f-1124a4c5c6fb">

### Steps to reproduce

1. install codegpt
2. goto "Settings | Tools | CodeGPT | Providers | Ollama (Local)"
3. set right host
4. apply
5. click "refresh models"
6. "Model" show "Loading" always ...

after debug and review code, i found commit 72abd4169f4dd2ab73c23ff28cf1b370af343b0f 
```kt
                        currentModel?.let {
                            if (models.contains(currentModel)) {
                                selectedItem = currentModel
                            } else {
                                OverlayUtil.showBalloon(
                                    format(
                                        CodeGPTBundle.get("validation.error.model.notExists"),
                                        currentModel
                                    ),
                                    MessageType.ERROR,
                                    modelComboBox
                                )
                            }
                        }
```

`currentModel` is default value `"Hit refresh to see models for this host"`
so `models.contains(currentModel)` will be false 

### CodeGPT version

2.8.2-241.1

### Operating System

macOS

**Comment by mprisznyak** [comment]
I confirm the bug. (Ubuntu)  I have installed CodeGPT from the JetBrains Marketplace.

**Comment by carlrobertoh** [comment]
Thank you for the feedback. This issue will be fixed in the next release.

**Comment by Yifei0727** [comment]
fixed 2.8.3-241.1 🎉

**Comment by rbsrcy** [comment]
> Thank you for the feedback. This issue will be fixed in the next release.

@carlrobertoh codeGPT 2.12.2-223 in idea 2023.3 also have the same problem,can help to fix it?
